src/Ztm.WebApi/Startup.cs

```diff
@@ -108,6 +137,7 @@ void ConfigureMvc(MvcOptions options)
             // Custom Model Binders.
             options.ModelBinderProviders.Insert(0, new BitcoinAddressModelBinderProvider());
             options.ModelBinderProviders.Insert(0, new PropertyAmountModelBinderProvider());
+            options.ModelMetadataDetailsProviders.Add(new SuppressChildValidationMetadataProvider(typeof(BitcoinAddress)));
```

Comment:
Why do we need to suppress validation for `BitcoinAddress`?

Reply:
This works around the infinity recursion issue after parse `BitcoinAddress`.

Reply:
I see. Better to put `FIXME` here so people can remove it once we upgrade the framework.

Reply:
Good idea